lib/element-properties.js

```diff
@@ -18,75 +18,39 @@ class ElementProperties {
         this.cy(this._cy);
     }
 
-    x(val) {
-        if (arguments.length === 0) {
+    setProperty(key, val) {
+        if (typeof val === 'undefined') {
```

Comment:
How come you use typeof and "undefined" when underneath you use the keyword undefined? Why not just use the keyword for both?

Reply:
Oh no reason in particular. I can make it more uniform.